src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -76,16 +76,31 @@ const LegacyGridItemConstrainedLarge = styled(GridItemConstrainedLarge)`
 
 const StyledSectionLabel = styled(SectionLabel)`
   margin-top: 0;
+  ${({ isRecommendation }) =>
```

Comment:
I'm not sure that this generic component should have a switch relating to its usage, should the switch relate to the fact it is used in the main column rather than the secondary column? Maybe there should be a `mainColumn` param to indicate it should layout differently when used in the main column? 

This would make it a little more generic for any future onward journeys integrated into the main column.